site/src/sphinx/auth.rst

```diff
@@ -0,0 +1,133 @@
+        ...
+    }
+
+Next, configure ``conf/shiro.ini`` based on your authentication system. A local database system of `Apache Shiro`_
+is used here to show you a simple example::
+
+    [users]
+    system-admin = system-admin-secret
+    dogma-admin = dogma-admin-secret
+
+.. note::
+
+    `Apache Shiro`_ supports RDBMS or LDAP based security system as well. You can find the example configuration
+    files under the ``conf/`` directory in the distribution.
+
+That's all. You are ready to use the security system of *Central Dogma*.
+
+Access Control
+--------------
+
+One of the way of configuring the access control system for *Central Dogma* is setting a project with a web UI.
+To open the web UI, you may access ``http://{your-central-dogma-domain-or-ip}:36462`` by your web browser.
+You may configure a project with HTTP APIs, but we recommend the web UI because it is easier and simple.
+
+Everyone who is logged in is able to create a new project, and he or she would be an owner of the project.
+If you have a right to configure a project, in other words, if you are an owner of the project,
+you can access the configuration UI of the project by clicking the pencil icon which is shown on the right
+of the project name.
+
+.. image:: _images/auth_1.png
+
+You can see the configuration UI for a project, in the below example it is for the project ``sample``.
+With this UI, you can add a user or token as a member of the project. You can also remove the member or token from the project.
+
+.. image:: _images/auth_2.png
+
+When you add a user as a member of the project, you need to choose a role of the member.
+There are 4 types of user role in the access control system of *Central Dogma*, however you can choose one
+of the ``Owner`` and ``Member`` role in the UI. More information about the role is as follows.
+
+- ``Administrator``
+
+  - the user that every permissions are assigned to. Only the administrator can restore removed project.
+    The administrators can be configured by ``conf/dogma.json`` as described the above.
+
+- ``Owner`` of a project
+
+  - the administrator of a project. The owners can create a new repository and remove a repository from the project.
+    They are also able to remove the project from the system and configure permissions for role, member and token.
+    As you expected, the administrators also can do that because they have every permission of the system.
+
+- ``Member`` of a project
+
+  - the user who can see the configuration of a project, but the members is not allowed to change the configuration.
+
+- ``Guest`` of a project
+
+  - the user who is logged in but he or she is neither an owner nor a member of a project. The user is not one of
+    the administrators, too. The guest is not allowed to see the configuration of a project.
+
+.. note::
+
+    Do not forget to make a new ``Application Token`` before adding a token to a project. ``Add a token`` button
+    would be disabled if there is no token.
```

Comment:
Should we show some message like 'Click here to create a new application token' when no token is available?